src/main/java/com/uber/cadence/internal/sync/WorkflowInternal.java

```diff
@@ -58,6 +58,7 @@
  * Never reference directly. It is public only because Java doesn't have internal package support.
  */
 public final class WorkflowInternal {
+  public static final int DEFAULT_VERSION = -1;
```

Comment:
let's make it 0

Reply:
Go uses -1 as default version so let's use the same here.

Reply:
We are Java ;)